lib/puppet/util/tagging.rb

```diff
@@ -47,9 +47,25 @@ def tag_if_valid(name)
     end
   end
 
-  # Is the receiver tagged with the given tags?
+  # Is the receiver tagged with at least one of the given tags?
```

Comment:
When rephrasing it is worth to update the text to say "Answers if the receiver is tagged with at least one of the given tags".